samples/identity/credentials/src/broker.ts

```diff
@@ -1,17 +1,18 @@
 import { SecretClient } from "@azure/keyvault-secrets";
+import { AuthenticationError } from "@azure/identity";
 
 // <BROKER>
-import { useIdentityPlugin, InteractiveBrowserCredential, AuthenticationError } from "@azure/identity";
+import { useIdentityPlugin, InteractiveBrowserCredential } from "@azure/identity";
 import { nativeBrokerPlugin } from "@azure/identity-broker";
 
 // Register the native broker plugin for brokered authentication
 useIdentityPlugin(nativeBrokerPlugin);
 
 // Use InteractiveBrowserCredential with broker for interactive authentication
 // On Windows: Uses Windows Authentication Manager (WAM) - you'll be prompted to sign in
-// On macOS/Linux: Opens a browser window for authentication
+// On macOS and Linux: Opens a browser window for authentication

```

Comment:
```suggestion
// On macOS: Opens a browser window for authentication, since the broker flow isn't currently supported.
// On Linux: Uses Microsoft Single Sign-on (SSO) for Linux.
```